src/WizardsWardrobeGui.lua

```diff
@@ -1895,3 +1899,44 @@ function WWG.StartAlphaAnimation( control, duration, startAlpha, endAlpha )
 	end
 	return animation, timeline
 end
+
+function WWG.ShowExitWarning()
```

Comment:
This function could either do nothing or hide the warnings, showing them is just one of three outcomes. Could be named better.